src/Compilers/Core/MSBuildTask/Microsoft.CSharp.Core.targets

```diff
@@ -47,6 +47,8 @@
 
       <!-- If we are targeting winmdobj we want to specifically the pdbFile property since we do not want it to collide with the output of winmdexp-->
       <PdbFile Condition="'$(PdbFile)' == '' AND '$(OutputType)' == 'winmdobj' AND '$(_DebugSymbolsProduced)' == 'true'">$(IntermediateOutputPath)$(TargetName).compile.pdb</PdbFile>
+
+      <Nullable Condition="'$(Nullable)' == '' AND '$(NullableContextOptions)' != ''">$(NullableContextOptions)</Nullable>
```

Comment:
Don't believe this will fix the issue. The core problem here is that Razor calls the Csc task directly. Hence in order to support their scenario we need to actually add both `Nullable` and `NullableContextOptions` to the `Csc` task itself. Changing the build file isn't actually necessary.

Reply:
Here is the problematic code that we need to account for 

https://github.com/aspnet/AspNetCore-Tooling/blob/81ea07caf7e5063b5f593d2d6a14d997643152e5/src/Razor/src/Microsoft.NET.Sdk.Razor/build/netstandard2.0/Microsoft.NET.Sdk.Razor.Compilation.targets#L156

Reply:
Indeed, thanks. I didn't want to modify `Csc` API as I thought that tends to be more durable and we want a temporary fix, but it looks like it's unavoidable.